server/build.gradle

```diff
@@ -54,17 +56,21 @@ dependencies {
     implementation "com.google.apis:google-api-services-gmail:v1-rev20200203-1.30.9"
     implementation "io.micronaut:micronaut-views"
     implementation "org.thymeleaf:thymeleaf:3.0.11.RELEASE"
+    implementation "io.micronaut.configuration:micronaut-hibernate-jpa"
+    implementation "io.micronaut.data:micronaut-data-jdbc:1.0.2"
```

Comment:
We should pull the µnaut data version into a gradle.properties file eventually so that these µnaut data module versions can be changed in one place. Not necessary to do in this in this PR.

Reply:
Agreed. I'll create a task in this sprint and make sure that it is changed.